jobs/integr8ly/w1-test-executor.yaml

```diff
@@ -50,6 +53,7 @@
                                     string(name: 'REPOSITORY', value: "${REPOSITORY}"),
                                     string(name: 'BRANCH', value: "${BRANCH}"),
                                     string(name: 'CLUSTER_URL', value: "${CLUSTER_URL}"),
+                                    string(name: 'NAMESPACE_PREFIX', value: "${NAMESPACE_PREFIX}"),
```

Comment:
Why do you add it only for AMQ address test?

Reply:
hmm I guess we don't use it in Fuse tests yet right?

Reply:
Because I thought it was not required for fuse-test. But I am debugging the failure of `fuse-test` and it seems that the namespace prefix needs to be added there as well. However, this will require code changes (MR to GitLab) and then followup PR here. So if you have no objections, I would merge this one and once the MR (I am working on it atm) is merged I can create the followup PR here.

Reply:
OK merging